invenio_base/wsgi.py

```diff
@@ -101,22 +132,31 @@ def wsgi_proxyfix(factory=None):
        use ``PROXYFIX_CONFIG`` instead.
     """
 
-    def create_wsgi(app, **kwargs):
-        wsgi_app = factory(app, **kwargs) if factory else app.wsgi_app
+    def create_wsgi(app: T) -> WSGICallable:
```

Comment:
was it intentionally that `**kwargs` has been dropped? from my own experiences, removing `**kwargs` can create bugs!

Reply:
good point. Il check this!